pkg/service/backup/README.md

```diff
@@ -0,0 +1,60 @@
+# Backup service
+
+### Useful links
+- official documentation of what the backup in scylla-manager is - https://manager.docs.scylladb.com/stable/backup/index.html
+- `sctool backup` command - https://manager.docs.scylladb.com/stable/sctool/backup.html
+- scylla manager agent config file (look for `Backup general configuration.`) - https://manager.docs.scylladb.com/stable/config/scylla-manager-agent-config.html
+- schylla manager config file (look for `Backup service configuration`) - https://manager.docs.scylladb.com/stable/config/scylla-manager-config.html
+
+### General picture 
```

Comment:
```suggestion
### Overview of backup processes
```

Reply:
I'll remove first picture from here and move it to scheduler service.